src/otx/algorithms/detection/adapters/mmdet/evaluation/evaluator.py

```diff
@@ -300,27 +300,21 @@ def evaluate_mask(self, results, logger, iou_thr):
         """
         assert len(results) == len(self.annotation[0]), "number of images should be equal!"
         num_imgs = len(results)
-        pool = Pool(self.nproc)  # pylint: disable=consider-using-with
         eval_results = []
+
+        ctx = mp.get_context("spawn")
         for class_id in range(self.num_classes):
             # get gt and det bboxes of this class
             cls_dets, cls_scores = self.get_mask_det_results(results, class_id)
             cls_gts = self.annotation[class_id]
 
             # compute tp and fp for each image with multiple processes
-            tpfpmiou = pool.starmap(
-                tpfpmiou_func, zip(cls_dets, cls_gts, cls_scores, [iou_thr for _ in range(num_imgs)])
-            )
+            with ctx.Pool(self.nproc) as p:
```

Comment:
Thanks for quick fix. How about to write `with ctx.Pool(self.nproc) as p:` outside the for statement to initialize Pool instance once? It can reduce a time, I think.